img/i/independentreserve.com.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 63.78 34.44"><path fill="#fff" d="M30.96 21.8c-.35 0-4.02-5.04-5.32-5.73-1.3-.68-3.28-.82-4.95-2.06l-2.26-1.45-4.56-2.86-7.72-4.54-3.32-2.13C1.83 2.39.89 1.67.01.88c-.12 2.39.8 4.95 2.16 6.97a16 16 0 0 0 5.63 4.99c2.66 1.5 7.68 2.29 10.53 5.14-3.07-.9-6.17-1.63-9.23-2.5-1-.29-2-.6-3-.91-.9-.27-1.76-.6-2.61-1 .3 1.53 1.53 2.96 2.87 4.12 1.17 1.01 3.68 1.8 5.28 2.27 2.36.67 6.93 1 9.1 2.54l-6.7-.43c-.8-.06-1.6-.14-2.4-.23-.7-.07-1.41-.18-2.1-.35.35 1.14 1.4 2.07 2.55 2.74 1.33.78 2.83 1 4.36 1.07 1.9.1 5.55-1.16 8.18.66-2.02.68-4.05 1.5-6 2.18l-1.74.57c-.51.17-1.03.3-1.56.4.42.7 1.31 1.03 2.22 1.16 1.07.15 2.15-.2 3.32-.4 2.36-.44 6.4-2.95 8.43-2.4v.05c-1.76 1.29-3.02 2.93-4.52 4.33-.32.3-.65.58-.99.87-.3.27-.62.5-.96.71.9.27 1.9.19 2.77-.03a5.1 5.1 0 0 0 2.4-1.27c.59-.6.9-1.3 1.22-1.98.24 1.54.84 3 1.75 4.26zM63.77 0c-.77.65-1.55 1.27-2.35 1.88h2.33c.03-.63.04-1.25.02-1.88m-5.33 4.96V7.9h2.64l.3-.43v-2.5ZM45.43 30.73c.96-.13 1.9-.49 2.32-1.22-.54-.1-1.08-.24-1.62-.4l-1.22-.4v2.05l.52-.02m-14.02 1.24v2.46l.15-.15c.28-.4.79-.86 1.23-2.3zm6.74-3.82h2.95v-.84l-2.95-1.05zm0-2.94v.94c.9-.6 1.9-.85 2.93-.94zm-3.38 4.01v2.3h2.26zm-1.61-1.38v-.05c.38-.09.78-.1 1.17-.04v-2.53h-2.92v2.93h2.12l-.37-.3zm4.99-6.45h2.95v-2.93h-2.95zm6.33-6.75v-2.42l-2.33 1.5c-.2.15-.4.28-.62.4v.52z"/><path fill="#005db2" d="M55.06 8.34v2.93h2.5l.45-.33v-2.6Zm-3.82 3.38H48.3v2.93h1.94l1-.37zm8.4 2.93a4.91 4.91 0 0 0 .51-1.37 18.75 18.75 0 0 1-1.7.68v.69zm-18.11 3.38h2.95v-2.94h-2.95Zm10.15-2.07v2.07h2.94v-2.87l-2.94.8zm-3.38 2.51v2.17l2.94-.6v-1.57zm-6.77 6.31h2.95v-2.33l-2.37.15a7 7 0 0 1 1.5-.75h-2.08z"/><path fill="#003a5a" d="M34.77 21.4h2.94v-2.93h-2.94zm10.14-2.93v2.93c.95-.27 1.96-.48 2.95-.67v-2.26zm-10.58 3.38h-2.86l-.06.03v2.9h2.92zm3.82 2.93h2.95v-2.93h-2.95zm12.22 0 .88-.43v-2.4l-2.14.2-.8.06v2.57zm-12.66 3.38v-2.94h-2.94v2.6c.4.09.81.2 1.25.34zm3.82.44v1.6l2.95.55V28.6Zm-6.76 4.3c.58.5 1.35.88 2.26 1.1l.68.14V32.2l-.23-.23h-2.7zM48.3 18.03h2.94v-1.95l-2.94.8zm-.44-3.38v-2.93h-2.58l-.36.23v2.7zm-6.76 3.38v-2.94h-1.9l-1.05.33v2.6z"/></svg>
```

Comment:
Maybe the logo with blue as the primary color is better?
https://www.independentreserve.com/_nuxt/img/symbol-blue.7294464.svg